test/polynomial_dfs.cpp

```diff
@@ -1293,4 +1296,75 @@ BOOST_AUTO_TEST_CASE(polynomial_dfs_zero_one_test) {
     BOOST_CHECK((small_poly - one * small_poly).is_zero());
 }
 
+BOOST_AUTO_TEST_CASE(polynomial_dfs_addition_perf_test, *boost::unit_test::disabled()) {
```

Comment:
I suppose we need some basic functional tests that are not disabled.

Reply:
Everything is already tested, there are lots of tests in this file that cover the new code.

These disabled tests will probably moved a separate "benchmark" sometime later.